### DISCUSSION TITLE
Code changes not reflected in app

### DISCUSSION THREAD

**Discussion by gabriellaputri** [quasarframework/quasar#15059] [CLI - Capacitor / Cordova modes]
My code changes is not reflected in the app when I open android emulator. I already run `quasar build` and `npx cap sync`. I also tried removing the www folder and re build it but still nothing changes. This is the dependencies that I use.
```
"@capacitor-community/fcm": "^2.0.2",
"@capacitor/android": "^3.8.0",
"@capacitor/app": "^1.1.1",
"@capacitor/core": "^3.8.0",
"@capacitor/ios": "^3.8.0",
"@capacitor/push-notifications": "^1.0.9",
"@capacitor/splash-screen": "^1.2.2",
"@capacitor/cli": "^3.8.0"
```
I also use Quasar v2.6.6.

**Comment by metalsadman** [comment]
uninstall old app, bump your app's version when building.
